cram_pr2/cram_pr2_environment_manipulation/src/prolog.lisp

```diff
@@ -0,0 +1,107 @@
+;;;
```

Comment:
This file should be renamed to something more meaningful, maybe designators-prolog.lisp or smt, or action-designators, ...

Reply:
I would say ``environment-manipulation-designators.lisp``